modules/administration-guide/partials/proc_migration-to-devworkspace-engine.adoc

```diff
@@ -0,0 +1,209 @@
+
+[id="migration-to-devworkspace-engine_{context}"]
+= Migration to {devworkspace} engine.
+
+This procedure describes how to migration to {devworkspace} engine using https://docs.openshift.com/container-platform/latest/operators/understanding/olm/olm-understanding-olm.html[OLM] to support the Devfile 2.0.0 file format and mentions how to do so on existing instances.
+
+.Prerequisites
+
+* The `{orch-cli}` tool is available.
+* An instance of {prod-short} deployed using xref:installation-guide:installing-che-on-openshift-4-using-operatorhub.adoc[Operator Hub] from `{prod-channel}` channel on OpenShift cluster version greater or equal to 4.8
+* OpenShift OAuth is enabled. See xref:configuring-openshift-oauth.adoc[].
+* Bundled PostgreSQL
+
+.Procedure
+
+. All workspaces must be stopped and changes pushed back to Git repositories.
```

Comment:
Is admin of Eclipse Che obligated to push back to Git repositories and then stop workspace?
Shouldn't workspace be stopped automatically?

Reply:
I am not aware of such possibility. This is a general recommendation about working workspaces.